packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -72,11 +72,7 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   // Connect evidence stream to advancer
   void observeIteration(subscribeEach(feedKit.public.getEvidenceStream()), {
     updateState(evidence) {
-      try {
-        advancer.handleTransactionEvent(evidence);
-      } catch (err) {
```

Comment:
we still should catch here. if an exception bubbles up to updateState then the observer will die.

Reply:
I had to remove this per type annotations since `handleTransactionEvent` catches all errors and does not re-throw them. If we want `handleTransactionEvent` to still bubble it, we can do that, but it seems like extra noise due to handled errors appearing in logs.

Reply:
> per type annotations

Type annotations don't specify what throws or doesn't … ?

> If we want handleTransactionEvent to still bubble it

No, I'm saying keep this defense. If `handleTransactionEvent` never throws, great. But if it does, this should catch it. It won't be any extra noise in the logs if it never happens.

Reply:
I misread the TS warning as this function will never throw (is that actually even a thing?). What was actually was an await or void since this is a promise. I've put this defense back.